src/index.js

```diff
@@ -30,6 +37,17 @@ export default class Undo {
     this.readOnly = configuration.readOnly;
     this.maxLength = maxLength || defaultOptions.maxLength;
     this.onUpdate = onUpdate || defaultOptions.onUpdate;
+    this.config = {
+      shortcuts: {
+        undo: defaultOptions.config.shortcuts.undo,
+        redo: defaultOptions.config.shortcuts.redo,
+      },
+    };
+
+    if (config && config.shortcuts) {
+      this.config.shortcuts.undo = config.shortcuts.undo;
+      this.config.shortcuts.redo = config.shortcuts.redo;
+    }
```

Comment:
@MarioRodriguezS what about moving this into the constructor itself, the control flow structure will not need anymore .... something like

```javascript
const { shortcuts } = config;
const defaultShortcuts = defaultOptions.config.shortcuts;

this.config = {
  shortcuts: {
    undo: shortcuts.undo || defaultShortcuts.undo,
    redo: shortcuts.redo || defaultShortcuts.redo,
  }
}
```

Reply:
@juanhurtado10 Thanks for your comment. The problem with your alternative is when the config is not passed in the undo instance, in that case config is undefined and an error will appear because you can not destructing an undefined.

Reply:
> @juanhurtado10 Thanks for your comment. The problem with your alternative is when the config is not passed in the undo instance, in that case config is undefined and an error will appear because you can not destructure an undefined.

@MarioRodriguezS I got you ... if you try setting a default value for the function param?